cerebras_tools_proxy.py

```
@@ -0,0 +1,562 @@
+    cerebras_status = "unknown"
+    try:
+        response = requests.get(
+            f"{CEREBRAS_BASE_URL}/models",
+            headers={"Authorization": f"Bearer {CEREBRAS_API_KEY}"},
+            timeout=5
+        )
+        if response.status_code in [200, 429]:  # 429 is OK, just rate limited
+            cerebras_status = "healthy"
+        else:
+            cerebras_status = f"error_{response.status_code}"
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)